chain/Cargo.toml

```diff
@@ -13,7 +13,8 @@ penumbra-tct = { path = "../tct" }
 
 # Penumbra dependencies
 decaf377 = { git = "https://github.com/penumbra-zone/decaf377" }
-jmt = "0.3"
+# We make JMT optional to support WASM for web extension.
+jmt = { version = "0.3", optional = true }
```

Comment:
Hold on, this makes the crate optional, but there's no other changes inside the library. Does that mean we weren't using it anyways, and could just remove it, or is something going to break when it's disabled?

Reply:
Hey, nice! You're right: we can fully remove `jmt` in both chain and crypto; still builds fine. Updated, CI is running now. Separately I'll scan for unused deps elsewhere as a housekeeping exercise.